imagetest/test_suites/network_perf/targets.txt

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
In general, to keep the scale of test runs more manageable, we should test the points when performance is expected to change. That being, the smallest machine shape at a given performance level. There may be a reason to also test performance with the largest machine shape in a series (TBD).

Reply:
Acknowledged. Will think of a way to do this efficiently.

Reply:
Done.